apps/launchpad/app/components/auth-cover.tsx

```diff
@@ -43,9 +45,9 @@ export function AuthCover({
             variant="primary"
             size="lg"
             onClick={login}
-            className="pointer-events-auto shadow-2xl min-w-[200px]"
+            className="pointer-events-auto shadow-2xl min-w-[220px]"
           >
-            Connect Wallet
+            Connect Wallet {actionText && `to ${actionText}`}
```

Comment:
Good idea! I made this original auth-cover component pretty quickly so it's nice to see ways you're extending it. We can always refactor to make more composable/flexible if needed too